app/assets/javascripts/charges.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
@schas002 sorry for this horrible pull request. It is my first pull request, once performed I did not know how to remove this empty file. I guess I should do a `git reset file`